cmd/application.go

```diff
@@ -16,9 +17,22 @@ func newApplication() *Application {
 	}
 
 	app.EnableMouse(cfg.Mouse)
+	app.SetInputCapture(onInputCapture)
 	return app
 }
 
+func onInputCapture(event *tcell.EventKey) *tcell.EventKey {
+ 	switch event.Name() {
+ 	case cfg.Keys.Quit:
+ 		app.Stop()
```

Comment:
Just send `Ctrl+C` (using tcell.NewEventKey) here instead of calling `Stop`.

Reply:
I'm not following. Do you mean that `Ctrl+C` should be emulated with tcell.NewEventKey? If so, no clue how to do that.